src/page.ts

```diff
@@ -111,7 +112,10 @@ export class Page extends platform.EventEmitter {
   _ownedContext: BrowserContext | undefined;
 
   constructor(delegate: PageDelegate, browserContext: BrowserContextBase) {
-    super();
+    super({
```

Comment:
Since we already inherit, perhaps use overridden methods instead?

Reply:
Sure.